### PR TITLE
Change folder name kubeless-test-client to kubeless integration

### PR DESCRIPTION
**Pull request by sayanh**
**Description**

Changes proposed in this pull request:

- Changed folder `kubeless-test-client` to `kubeless-integration`
- Enabled kubeless integration tests to just run on clusters

**Related issue(s)**

[#1620](https://github.com/kyma-project/kyma/issues/1620)

### CODE REVIEW

File: resources/core/charts/kubeless/templates/tests/test-kubeless.yaml [hunk 1]
```diff
@@ -73,6 +73,7 @@ subjects:
   name: test-{{ template "fullname" . }}
   namespace: {{ .Release.Namespace }}
 ---
+{{- if not .Values.global.isLocalEnv }}
```
Comment (rakesh-garimella) [inline resources/core/charts/kubeless/templates/tests/test-kubeless.yaml]: should we skip the complete file ?  We can skip the deployment of cluster role too ?

Comment (sayanh) [reply resources/core/charts/kubeless/templates/tests/test-kubeless.yaml]: We can skip the deployment of `clusterrole` for now but we will introduce it again in this [PR](https://github.com/kyma-project/kyma/pull/1754) as creation/deletion of namespace will be done.  This PR is targetted only to have aesthetic changes and keep the existing functionality as is.

File: tests/kubeless-integration/README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 ## Overview
 
-The `kubeless-test-client` contains the following:
+The `kubeless-integration` tests kubeless along with other Kyma resources. It contains the following:
```
Comment (klaudiagrz) [inline tests/kubeless-integration/README.md]:
```suggestion
The `kubeless-integration` test client tests Kubeless along with other Kyma resources. It contains the following:
```

Comment (klaudiagrz) [inline tests/kubeless-integration/README.md]:
```suggestion
The `kubeless-integration` folder contains tests for Kubeless integration with other Kyma resources. It contains the following:
```

**Pull request merged**